docs/README.md

```diff
@@ -0,0 +1,240 @@
+{
+  "comment":"/**\n * A Label to be used for Nodes or Edges.\n * /",
+  "meta":{
+    "range":[3770,41303],
+    "filename":"Label.js",
+    "lineno":167,
+    "columnno":0,
+    "path":"/home/wim/projects/github/vis/lib/network/modules/components/shared",
+    "code":{
+      "id":"astnode100065034",
+      "name":"Label",
+      "type":"ClassDeclaration",
+      "paramnames":["body","options","edgelabel"]
+    }
+  },
+  "classdesc":"
+A Label to be used for Nodes or Edges.
+
+",
+  "name":"Label",
+  "longname":"Label",
+  "kind":"class",
+  "scope":"global",
+  "params":[
+    {"type":{"names":["Object"]},"name":"body"},
+    {"type":{"names":["Object"]},"name":"options"},
+    {"type":{"names":["boolean"]},"optional":true,"defaultvalue":false,"name":"edgelabel"}
+  ],
+  "___id":"T000002R005289",
+  "___s":true
+}
+```
+
+  Example of item for an instance method.
+
+```
+    var tmp = data().filter({name:'_drawText'}).get();
+```
+
+  Full output returned:
+
+```
+[{
+  "comment":"/**\n *\n * @param {CanvasRenderingContext2D} ctx\n * @param {boolean} selected\n * @param {boolean} hover\n * @param {number} x\n * @param {number} y\n * @param {string} [baseline='middle']\n * @private\n * /",
+  "meta":{
+    "range":[20060,22269],
+    "filename":"Label.js",
+    "lineno":652,
+    "columnno":2,
+    "path":"/home/wim/projects/github/vis/lib/network/modules/components/shared",
+    "code":{
+      "id":"astnode100066427",
+      "name":"Label#_drawText",
+      "type":"MethodDefinition",
+      "paramnames":["ctx","selected","hover","x","y","baseline"]
+    },
+    "vars":{"":null}
+  },
+  "params":[
+    {"type":{"names":["CanvasRenderingContext2D"]},"name":"ctx"},
+    {"type":{"names":["boolean"]},"name":"selected"},
+    {"type":{"names":["boolean"]},"name":"hover"},
+    {"type":{"names":["number"]},"name":"x"},
+    {"type":{"names":["number"]},"name":"y"},
+    {"type":{"names":["string"]},"optional":true,"defaultvalue":"'middle'","name":"baseline"}
+  ],
+  "access":"private",
+  "name":"_drawText",
+  "longname":"Label#_drawText",
+  "kind":"function",
+  "memberof":"Label",
+  "scope":"instance",
+  "___id":"T000002R005388",
+  "___s":true
+}]
+```
+
+  ## `jsdoc` template rendering
+
+  See `function createRenderer(fromDir, data)` in code for usage.
+
+  There are two calls for rendering templates:
+ 
+  - `var html = renderer.render(inFile, docData);`
+  - `var html = renderer.partial(inFile, docData);`
+ 
+  The difference is that `render()` will use a default layout template, if present, which
+  will encapsulate all html. This can be set by:
+ 
+  ```
+    renderer.layout = 'path/to/default/layout.tmpl'; 
+  ```
+ 
+  Parameter `docData` is a hash which is used to pass parameters into a template.
+  The standard way of using this appear to be:
+  ```
+  <?js
+    var data = obj;   // Whatever docData is
+    var self = this;
+  ?>
+  ```
+ 
+  But it also appear to be possible to use the elements of docData directly:
+  ```
```

Comment:
You can get proper syntax highlighting by adding the name to your code block.

ie, 
<pre>
```javascript
var docData =...
```
<pre>

Reply:
json is also a valid key word for syntax highlighting

Reply:
yeah, I knew that, it just didn't occur to me to use it here.

```js
 var msg = 'js works as well.';
```

<pre>And now I learnt from you how to display ```js itself!</pre>

Reply:
Please do keep on reviewing! And yes try it out, please, if only to see if it works on machines other than mine.